mesh/mesh-io-generic.c

```diff
@@ -177,6 +177,66 @@ static void event_callback(const void *buf, uint8_t size, void *user_data)
 	}
 }
 
+static void configure_hci(struct mesh_io_private *io)
+{
+	struct bt_hci_cmd_le_set_scan_parameters cmd;
+	struct bt_hci_cmd_set_event_mask cmd_sem;
+	struct bt_hci_cmd_le_set_event_mask cmd_slem;
+
+	/* Set scan parameters */
+	cmd.type=0x01; /* Active scanning. Scanning PDUs may be sent. */
+	cmd.interval=0x0010; /* Scan Interval = N * 0.625ms */
+	cmd.window=0x0010; /* Scan Window = N * 0.625ms */
```

Comment:
Based on recent experimentation you actually get much better performance with 30ms scan interval and window (at least with Zephyr controller), so I'd recommend changing to that.

Reply:
Ok, will change.